### DISCUSSION TITLE
Max Time Duration to move a notice from Submitted --> Published or Not Published

### DISCUSSION THREAD

**Discussion by ravesingh1983** [OP-TED/eForms-SDK#325] [Q&A] [answered]
In case of Preferred Publication as ASAP what is the maximum wait time to move notice from Submitted to Published/ Not Published State

**Comment by ferrakaop** [comment, marked as answer]
This is not a technical question and it should be addressed to the TED helpdesk at ted@publications.europa.eu

An overview of the lifecycle of eForms notices was also covered in the workshop of 1 February 2023 - see https://op.europa.eu/en/web/ted-eforms/ - you'll see that the procurement directives require that the Publications Office publish submitted notices within 5 calendar days.
